.gitignore

```diff
@@ -1,2 +1,22 @@
-/.vscode
-node_modules/
+.DS_Store
+node_modules
+/dist
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
+.vscode
```

Comment:
@avishj I see `.vscode` here but still I can see the `.vscode` directory in the repo. Please delete the `.vscode folder from the repo. Ideally we shouldn't push the local editor settings.

Reply:
Ok, missed that one, will fix it

Reply:
@vinitshahdeo I had a look at the repo through File Explorer, couldn't find the .vscode directory, could you please tell me the location of that directory?

Reply:
`.vscode` directory remain hidden. Check for hidden files and folders.

Reply:
@avishj you can find .vscode on your code editor like VS Code too.